__tests__/installer.test.ts

```diff
@@ -6,11 +6,12 @@ import cp from 'child_process';
 import osm = require('os');
 import path from 'path';
 import * as main from '../src/main';
-import * as im from '../src/installer';
 import * as auth from '../src/authutil';
-
 let nodeTestManifest = require('./data/versions-manifest.json');
 let nodeTestDist = require('./data/node-dist-index.json');
+import * as nv from '../src/installer';
```

Comment:
Not sure why this was changed 🤔, I think it should be changed back to the original name `im` .  The current one doesn't reflect the file it was imported from.

Reply:
nopw its fine as nv